automation/docker/application.conf.ctmpl

```diff
@@ -29,7 +29,7 @@ fireCloud {
   fireCloudId = "{{$secrets.Data.firecloud_id}}"
 
   # TODO more Config class nonsense
-  tcgaAuthDomain = “TCGA-dbGaP-Authorized”
+  tcgaAuthDomain = "TCGA-dbGaP-Authorized"
```

Comment:
The quotes were weird here for some reason - not sure how that happened.

Reply:
someone almost certainly copied from a google/word doc